k8s/prod/common-overlay/financial-remedy/kustomization.yaml

```diff
@@ -3,3 +3,6 @@ kind: Kustomization
 namespace: financial-remedy
 bases:
 - ../../../namespaces/financial-remedy
+- ../../../namespaces/financial-remedy/finrem-ns/finrem-ns.yaml
```

Comment:
normally you would add this to the common base at this point rather than adding it to the prod kustomization